modules/core/05-port/types/module.go

```diff
@@ -110,7 +111,11 @@ type ICS4Wrapper interface {
 	SendPacket(
 		ctx sdk.Context,
 		chanCap *capabilitytypes.Capability,
-		packet exported.PacketI,
+		srcPort string,
+		srcChannel string,
```

Comment:
My preference is `sourcePort` `sourceChannel` since there isn't a reason not to be explicit